app/views/subnets/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Subnet|Name") %></th>
-      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network") %></th>
+      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network Address") %></th>
       <th class="col-md-3"><%= _("Domains") %></th>
```

Comment:
When reviewing the page, it is "Network address",  but in the coding it is "Network". and I added "Address" after Network.
If it is wrong, please correct me. Thanks.